distribution/docs/src/main/jdocs/content/_quickstart/quickstart-installing-contents.adoc

```diff
@@ -0,0 +1,153 @@
+:title: Installing (Quick Start)
+:type: quickStart
+:status: published
+:summary: Installation of an example instance.
+:order: 00
+
+This quick tutorial will enable install, configuring and using a basic instance of ${branding}.
+
+[NOTE]
+====
+This tutorial is intended for setting up a test, demonstration, or trial installation of ${branding}.
+For more complete installation and configuration steps, see <<_installing,Installing>>.
+====
+
+These steps will demonstrate:
+
+- [*] <<_quick_install_prerequisites,Prerequisites>>.
+- [*] <<_quick_install_of_${branding-lowercase},Quick Install of ${branding}>>.
+- [*] <<_ingesting_sample_data,Ingesting Data>>.
+
+=== Quick Install Prerequisites
+
+These are the basic requirements to set up the environment to run a ${branding}.
+
+==== Hardware Requirements (Quick Install)
+
+* At least 4096MB of memory for ${branding}.
+** This amount can be increased to support memory-intensive applications. See <<jvm-memory-configuration, Memory Considerations>>.
+
+==== Java Requirements (Quick Install)
+
+Set up Java to run ${branding}.
+
+* Install/Upgrade to Java 8 http://www.oracle.com/technetwork/java/javase/downloads/index.html[J2SE 8 SDK]
+** The recommended version is http://www.oracle.com/technetwork/java/javase/8u60-relnotes-2620227.html[8u60] or later.
+** Java Version and Build numbers must contain only number values.
+* Supported platforms are *NIX - Unix/Linux/OSX, Solaris, and Windows.
```

Comment:
I would nix the *NIX (ha-ha). How about:

> Supported platforms are Microsoft Windows, Linux, Solaris, and macOS.

Little known fact: Apple change the name of their desktop/laptop operating system to "macOS" last year.

The term Unix can be really broad at times, so I would recommend that we not advertise our support unless people have successfully fielded a DIB with the OS in question. Also, Solaris is a kind of Unix. 

I reordered the terms in what _I think_ is their frequency in the real-world.